PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -238,59 +238,64 @@ Usage: PreeFreeSurferPipeline.sh [options]
              or={i,j,NONE}          of the spin echo field map. 
                                     (Only applies when using a spin echo field map.)
   --t1samplespacing=<seconds>       T1 image sample spacing, "NONE" if not used
   --t2samplespacing=<seconds>       T2 image sample spacing, "NONE" if not used
   --unwarpdir={x,y,z,x-,y-,z-}      Readout direction of the T1w and T2w images (according to the *voxel axes)
            or={i,j,k,i-,j-,k-}      (Used with either a gradient echo field map 
-									or a spin echo field map)
+                                    or a spin echo field map)
   --gdcoeffs=<file path>            File containing gradient distortion
-									coefficients, Set to "NONE" to turn off
+                                    coefficients, Set to "NONE" to turn off
   --avgrdcmethod=<avgrdcmethod>     Averaging and readout distortion correction
-									method. See below for supported values.
+                                    method. See below for supported values.
+  --custombrain=(NONE|MASK|CUSTOM)  Whether to use a custom brain mask (MASK) the 
+                                    pre-existing custom brain images (CUSTOM)
+                                    or not (NONE)
 
-	  "${NONE_METHOD_OPT}"
-		 average any repeats with no readout distortion correction
+    "${NONE_METHOD_OPT}"
+     average any repeats with no readout distortion correction
 
-	  "${FIELDMAP_METHOD_OPT}"
-		 equivalent to "${SIEMENS_METHOD_OPT}" (see below)
-		 SiemensFieldMap is preferred. This option value is maintained for
-		 backward compatibility.
+    "${FIELDMAP_METHOD_OPT}"
+       equivalent to "${SIEMENS_METHOD_OPT}" (see below)
+       SiemensFieldMap is preferred. This option value is maintained for
+       backward compatibility.
 
-	  "${SPIN_ECHO_METHOD_OPT}"
-		 average any repeats and use Spin Echo Field Maps for readout
-		 distortion correction
+    "${SPIN_ECHO_METHOD_OPT}"
+       average any repeats and use Spin Echo Field Maps for readout
+       distortion correction
 
-	  "${GENERAL_ELECTRIC_METHOD_OPT}"
-		 average any repeats and use General Electric specific Gradient
-		 Echo Field Maps for readout distortion correction
+    "${GENERAL_ELECTRIC_METHOD_OPT}"
+       average any repeats and use General Electric specific Gradient
+       Echo Field Maps for readout distortion correction
 
-	  "${SIEMENS_METHOD_OPT}"
-		 average any repeats and use Siemens specific Gradient Echo
-		 Field Maps for readout distortion correction
+    "${SIEMENS_METHOD_OPT}"
+       average any repeats and use Siemens specific Gradient Echo
+       Field Maps for readout distortion correction
 
   --topupconfig=<file path>      Configuration file for topup or "NONE" if not
-								 used
+                 used
   --bfsigma=<value>              Bias Field Smoothing Sigma (optional)
+  --t1biascorrect=(YES|NONE)     Bias Field Correction for T1w image only (YES or NONE if not, YES default; optional)
```

Comment:
Rather than framing this around a "T1 only" bias correction option, perhaps we should just have a more general `--biascorrect` flag that is (YES|NO|NONE) (with NO|NONE doing the same thing).  Then, add a parameter checking step that would generate an error if someone specified `--biascorrect=NO` but they provided a T2w input image.

Reply:
What if the intention of the use would be to omit bias correction even when t2w is provided?

Reply:
I think we have three options:

1/ A clearer definition:
```
  [--t1biascorrect=(YES|NO|NONE)]     Whether to run Bias Field Correction even when only T1w image is present (YES default; optional)
```
2/ Renaming to `--biascorrect`, more detailed description, checking if T2w is present and aborting with an explanation if it is.
```
  [--biascorrect=(YES|NO|NONE)]       Whether to run (YES) or omit (NO, NONE) Bias Field Correction 
                                      in cases when only T1w image is present. The setting NO or NONE is invalid
                                      when T2w image is present. (YES default; optional)
```

3/ Renaming to `--biascorrect` and making actual use of it. In this scenarion it fails check for mpp-mode 'standard' and it results in no bias field correction even if T2w image is present. This option would though require additional coding and would result in more inappropriately named 'dummy' files.

I vote for 1/ :-)

Reply:
2 is what I was indeed proposing, and what I vote for.

Reply:
I think I prefer 1.  I don't really want the bias correct turned off if a T2w is present and I think focusing on the T1w bias correction is clearer.

Reply:
The parameter was excluded and Bias Field Correction is allways performed.

Reply:
Having both NO and NONE do the same thing, which is to make it change depending on the provided inputs, seems inconsistent to me.  Intuitively, specifying `--biascorrect=NO` and getting bias correction (because you have a T2w) seems confusing.

If the default is to always do bias correction, and the user is editing the inputs anyway, it would be less confusing to just do what it looks like: `--biascorrect=NO` turns off bias correction, even when you have a T2w (and such a setting is not hcp-compliant).  If it is wrong or complicated to turn it off when both are present, then that can be an error, and then the option doesn't need to claim to change behavior.

As for `NONE`, I have never liked it, but it is only meant as a placeholder for a *filename* when we have wanted to omit an input (because some people didn't want to learn how quotes work).  It should not be accepted on any YES/NO option, and in my opinion it should be used sparingly, if at all, going forward.

Reply:
@coalsont in the latest version of the PR the option is excluded and the BFC is always performed.

Reply:
The options are not YES/NO or NONE as a placeholder for absent file or folder.